specs/windows/interface_details.table

```diff
@@ -0,0 +1,33 @@
+table_name("interface_details")
```

Comment:
This is overloading `interface_details`: https://osquery.io/docs/tables/#interface_details similar to `apps` we cannot support differing specifications per-platform. We either merge, with a bias towards not changing the existing column names/types or create a new table.